components/scream/src/physics/share/tests/physics_unit_tests_common.hpp

```diff
@@ -0,0 +1,65 @@
+#ifndef PHYSICS_UNIT_TESTS_COMMON_HPP
```

Comment:
should this file be named share instead of common? 8-)

Reply:
Yes.

Reply:
Actually, no. The COMMON part comes from the fact that these constructs are provided as a common element to all physics/share unit tests. This mirrors src/physics/p3/tests/p3_unit_tests_common.hpp, for example.

Reply:
Naming things is hard.

Reply:
Yeah, I hear you. Ok, I'm fine w/ keeping common.